test/user.test.js

```diff
@@ -254,27 +254,31 @@ describe('User', function() {
     });
 
     it('invalidates the user\'s accessToken when the user is deleted all', function(done) {
-      var usersId, accessTokenId;
+      var usersId = [];
+      var accessTokenId;
       async.series([
         function(next) {
-          User.create([{ name: 'myname', email: 'b@c.com', password: 'bar' },
-          { name: 'myname', email: 'd@c.com', password: 'bar' }], function(err, user) {
-            usersId = user.id;
+          User.create([
+          { name: 'myname', email: 'b@c.com', password: 'bar' },
+          { name: 'myname', email: 'd@c.com', password: 'bar' },
```

Comment:
indent one more level for the objects (is this using the latest eslint-config-loopback@4.0.0)? I thought we removed the spaces at the front and back.